vitals/BoundryRules.java

```diff
@@ -0,0 +1,62 @@
+package vitals;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BoundryRules {
+
+    static Map<String, Map<String, String>> rules;
+
+    public static void setConditions() {
+        rules = new HashMap<>();
+        Map<String, String> soc = new HashMap<>();
+        soc.put("0-20", "LOW_SOC_BREACH");
```

Comment:
How will negative numbers be represented in this string? Will the `-` not interfere with the parsing?
You don't need to design outside the requirements. But avoid adding new 'implicit' assumptions that make it hard to add future flexibility.